pages/news/2020-XX-XX-new-in-2-3-rekordbox-support.html

```diff
@@ -0,0 +1,58 @@
+title: New in 2.3: Importing tracks and cues from Rekordbox removable devices
+author: ehendrikd
+date: 2020-07-13 20:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Do you rock your gigs armed only with your cans and a Rekordbox prepared USB flash drive? Ever been tempted to play your carefully curated tracks on something other than a CDJ or Rekordbox, on something that is free and open-source? We have news for you!
+
+Mixxx 2.3 will support reading the following from Rekordbox prepared removable devices on all of Mixxx's supported platforms:
+
+* Folders
+* Playlists
+* Beatgrids
+* Hotcues
+* Memory cues
+* Loops
+
+You can use this new feature now in [Mixxx 2.3 beta](/download/#unstable). This includes importing the key and BPM analyzed by Rekordbox, and comments and colors for tracks, cues and loops. **We have to be clear however:** this feature *only* reads Rekordbox prepared removable devices, such as USB flash drives or external hard drives. It *will not* read your locally stored Rekordbox collection if you also have Rekordbox installed. The main Rekordbox database (both for Rekordbox 5 and Rekordbox 6) use completely different formats from the USB drives, so reading those databases would be a whole other project.
+
+## Importing from Rekordbox to Mixxx
+
+After you have exported music to a USB drive in Rekordbox, simply plug it in while Mixxx is running. Click on Rekordbox in the library, and all USB drives prepared with Rekordbox will be visible (attach as many as you like!). Then click on the desired removable device, and all your folders and playlist that you have so tirelessly prepared will be revealed for you to play from.
+
+<div style="text-align: center;"><img src="{% static '/static/images/news/mixxx-rekordbox-usb.png' %}" /></div>
```

Comment:
Please use markdown images and provide an ALT name in square brackets:

```suggestion
![Mixxx showing a Rekordbox USB library]({% static '/static/images/news/mixxx-rekordbox-usb.png' %}) 
```

Also applies to the other images in this post.